test/MaxBTCCore.test.sol

```diff
@@ -199,11 +201,20 @@ contract MaxBTCCoreTest is Test {
 
     function testMintFeeByCollector() external {
         uint256 amount = 3e7;
+        provider.publishAum(int256(1), depositToken.decimals());
         vm.prank(FEE_COLLECTOR);
         core.mintFee(amount);
         assertEq(maxbtc.balanceOf(FEE_COLLECTOR), amount, "fee minted");
     }
 
+    function testMintFeeRevertsWhenAumNonPositive() external {
+        uint256 amount = 1e7;
+        // do not publish AUM (default 0) -> should revert
```

Comment:
I think we better call `provider.publishAum(int256(1), depositToken.decimals());` in the set up function, and here override it with `provider.publishAum(int256(-1), depositToken.decimals());`.

This way, we won't have to manually specify AUM in `testMintFeeByCollector()` and in other potentional future tests, too